sdk/python/foundation-models/system/inference/zero-shot-image-classification/zero-shot-image-classification-batch-endpoint.ipynb

```diff
@@ -236,9 +236,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 3.2 Prepare CSV file with base64 images for batch inference input\n",
+    "#### 3.2 Prepare a folder of CSV files with base64 images for batch inference input\n",
     "\n",
-    "We can provide input images to batch inference in a csv file containing a column named \"image\" having images in base64 format. We can provide the candidate labels in the same csv file in the first row of a column named \"text\"."
+    "We can provide input images to batch inference in a csv file containing a column named \"image\" having images in base64 format. We can provide the candidate labels in the same csv file in the first row of a column named \"text\".\n",
+    "Each CSV file is a mini-batch, so we split up the datasets into multiple CSV files."
```

Comment:
Should we update this comment to some like below (Please feel free to add a shortened version of this)
"`mini_batch_size` input in `Create a batch deployment` step below controls the number of csv files processed at once by the model. To limit the number of images processed in each mini batch, we split up the dataset into multiple csv files each with 10 images."